aiogram/dispatcher/filters/builtin.py

```diff
@@ -757,3 +757,72 @@ def __init__(self, is_media_group: bool):
 
     async def check(self, message: types.Message) -> bool:
         return bool(getattr(message, "media_group_id")) is self.is_media_group
+
+
+class StorageDataFilter(BoundFilter):
+    """
+    Check if all items matches the relevant items in the current storage data.
+    """
```

Comment:
Please, add examples to:
1. `examples` folder
2. `StorageDataFilter` docstrings

Also don't forget to add description of asterisk behaviour